Add sequence for BBF10K_000464

noahsprent opened the pull request:
I've added the sequence for BBF10K_000464 that was missing (see #107). The whole excel file was actually blank on the open-yeast-collection branch so I edited the file from develop with the new sequence. Looks like it failed the checks so not sure if I broke something that @jakebeal is working on?

jakebeal commented:
I needed to change the data source imports: check out the data source columns in https://github.com/iGEM-Engineering/iGEM-distribution/blob/5a9dc82b3c48d1c6ebecfe908f556c1acc3ca3cf/Open%20Yeast%20Collection/Open%20Yeast%20Collection.xlsx

noahsprent commented:
Ok yes I was being stupid, I opened the excel file and thought it was empty but I was in the wrong sheet 🤦. I've added the sequence to the new excel file that you made and changed the design notes to "Taken from open yeast collection" and left the data source prefix blank as you did with other parts like this. Thanks!
